documentation/doxygen/Makefile

```
@@ -16,11 +16,13 @@ export DOXYGEN_NOTEBOOK_PATH := $(DOXYGEN_OUTPUT_DIRECTORY)/notebooks
 export DOXYGEN_STRIP_PATH := $(shell cd $(PWD)/../..; pwd)
 export DOXYGEN_INCLUDE_PATH := $(shell find $(DOXYGEN_STRIP_PATH) -type d -name dictpch -prune -o -type d -name inc)
 
+PYZ_DIR = ../../bindings/pyroot/pythonizations/python/ROOT/pythonization
+
 define MkDir
 	+@[ -d $1 ] || mkdir -p $1
 endef
 
-all: filter folders mathjax images doxygen
+all: filter folders mathjax images pyzdoc doxygen
```

Not really necessary. `doxygen` already has the dependency.
```suggestion
all: filter folders mathjax images doxygen
```

---

True!